Libraries/Lists/__tests__/__snapshots__/VirtualizedList-test.js.snap

```diff
@@ -1993,45 +1954,10 @@ exports[`discards intitial render if initialScrollIndex != 0 1`] = `
     <View
       style={
         Object {
-          "height": 90,
+          "height": 140,
         }
       }
     />
-    <View
```

Comment:
There are some cases like this, where we render less, but the new behavior looks more correct to me given inputs.